tests/testcases/manual.json

```diff
@@ -89,7 +89,7 @@
     "name": "completeTransferExpert",
     "tx": {
       "account_number": "0",
-      "chain_id": "cosmoshub-3",
+      "chain_id": "comsoshub-4",
```

Comment:
There is a typo here

Reply:
well.. almost everywhere :)

Reply:
This makes me wonder how we didn't get it Zemu

Reply:
Because the type was also in the app haha...fixed now!